Reports/router.js

```diff
@@ -1,4 +1,38 @@
 var express = require('express');
+var mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost:27017');
```

Comment:
I am a fan of a [single source of truth](https://en.wikipedia.org/wiki/Single_source_of_truth). In this case the port would be specified in a `.config` or global variable to conform to that philosophy.